html/files/tagline/tagline-nl.txt

```diff
@@ -1,15 +1,15 @@
-message:Discover the Nutri-Score
-url:https://world.openfoodfacts.org/nutriscore
+bericht:Ontdek Nutri-Score
```

Comment:
Is the prefix supposed to be translated? I don't know what format the `tagline` txs are. 😄

Reply:
no they are not, and I think now that taxonomies are translatable, we'd be better off using taxonomies for it